### PR TITLE
Switch to use keys for node urls

### PR DESCRIPTION
**Pull request by yozuysal**
Switch from hardcoded urls to the ones from config for node urls. 

### How was this tested?
- [X] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### Manual test cases
Enable wallet and test node related functionality

### CODE REVIEW

File: WalletCore/CryptoConfig.swift [hunk 1]
```diff
@@ -62,17 +62,23 @@ public enum EthNode: String, CaseIterable, Identifiable {
...
         case .Polygon:
-            return "https://polygon-mainnet.infura.io/v3/25b0f8c5b9004da38236104927e630b5"
+            return "CRYPTO_POLYGON_URL"
         }
     }
 
+
+
+    public var url: URL {
+        URL(string: Bundle.main.object(forInfoDictionaryKey: configKey) as! String)!
```
Comment (chung-lai) [inline WalletCore/CryptoConfig.swift:79]: Given not all devs are going to set up these api keys in their environment. We should return `URL?` and remove `!`

Comment (yozuysal) [reply WalletCore/CryptoConfig.swift]: that is a great point :)

It did require a bit more surgery to use URL? here though. PTAL

File: WalletCore/CryptoConfig.swift [hunk 2]
```diff
@@ -112,58 +114,24 @@ public class CryptoConfig {
         return "My Neeva Crypto Wallet"
     }
 
-    public func sendEth(amount: String, sendToAccountAddress: String, completion: () -> Void) {
-        do {
```
Comment (chung-lai) [inline WalletCore/CryptoConfig.swift:116]: I assume these are all just clean up? Not sure how this relate to changing to `URL?` :P

**Review by chung-lai — APPROVED** [review]
Just one comment about the refactoring change. otherwise, LGTM!

File: WalletCore/CryptoConfig.swift (hunk 2 above)
Comment (yozuysal) [reply WalletCore/CryptoConfig.swift:116]: a nil URL creates a nil web3 object and the path that I am calling now is guarded against having a nil web3 (it refreshes it)

**Pull request merged**